firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigAutoFetch.java

```diff
@@ -119,7 +119,8 @@ private void handleNotifications(InputStream inputStream) throws IOException {
     BufferedReader reader = new BufferedReader((new InputStreamReader(inputStream, "utf-8")));
     String message;
     while ((message = reader.readLine()) != null) {
-      if (!message.contains("latestTemplateVersionNumber")) {
+      if (!message.contains("latestTemplateVersionNumber")
+          && !message.contains("featureDisabled")) {
```

Comment:
Without more context I'd assume that the `featureDisabled` would be checked first to short-cut the rest of the cut. Why is it checked last?

Reply:
I added the one pager for this change to the description. This flag will always be in the response so this check skips any other messages we may get; it doesn't matter the order of the check.

Reply:
Also added tests.